manual_test_functions.py

```diff
@@ -2,40 +2,60 @@
 
 Intended as a simple end-to-end test for assurance when making changes.
 Uses all existing group, event, message `get_` methods.
-Doesn't yet use `get_person(id)` or any `send_`, `update_` methods."""
+Doesn't yet use `get_person(user)` or any `send_`, `update_` methods."""
 
 import asyncio
 import random
 
 from config import password, username
 from spond import spond
 
+DUMMY_ID = "DUMMY_ID"
+
 
 async def main() -> None:
     s = spond.Spond(username=username, password=password)
 
-    print("Getting all groups...")
+    # GROUPS
+
+    print("\nGetting all groups...")
```

Comment:
If you want an extra newline at the end, use `print("blah blah blah", end="\n\n")`?

Reply:
Looks like I resolved this without committing my comment... 🤦

It's a newline at the beginning, not end. But I think this is too minor to worry about in a script that only exists as an interim measure, indeed this PR goes some way to eliminating it.